src/main/thrift/parquet.thrift

```diff
@@ -547,6 +547,58 @@ struct RowGroup {
   4: optional list<SortingColumn> sorting_columns
 }
 
+/** Identifier for built-in sort order used to produce min and max values. */
+enum Order {
+  /**
+   * The signed ordering is the order produced by comparing single primitive
+   * values with a signed comparator, or the lexicographic ordering produced by
+   * comparing each byte of a binary or fixed using a signed comparator.
+   *
+   * (A signed comparator uses the most-significant bit as a sign bit; an
+   * unsigned comparator uses the most-significant bit as part of the value's
+   * magnitude. Note that unsigned comparison is not defined for floating
+   * point values.)
+   */
+  SIGNED = 0;
+
+  /**
+   * The unsigned ordering is produced by comparing single primitive values
+   * with an unsigned comparison, or the lexicographic ordering produced by
+   * comparing each byte of a binary or fixed using an unsigned comparator.
+   *
+   * (A signed comparator uses the most-significant bit as a sign bit; an
+   * unsigned comparator uses the most-significant bit as part of the value's
+   * magnitude. Note that unsigned comparison is not defined for floating
+   * point values.)
+   */
+  UNSIGNED = 1;
+
+  /**
+   * Identifiers for custom orderings, to be defined in the ColumnOrder struct.
+   */
+  //CUSTOM = 2;
```

Comment:
I don't understand the claim that INT96 aren't interpretable as integers. They don't map to a common hardware integer type but they're still logically a twos-complement integer.

I think on the Impala side we may end up implementing min-max stats for INT96 ahead of doing a larger rework of the timestamp data type, so it would be good to have clarity about what other implementations should do when encountered with INT96 min-max stats.

My take is that they should either ignore INT96 stats (if they don't want to implement a deprecated type) or use signed ordering. Parquet-mr doesn't do this currently. Does everyone agree that https://issues.apache.org/jira/browse/PARQUET-840 is a bug in parquet-mr?